src/Tizen.NUI.Components/Controls/Control.cs

```diff
@@ -95,9 +115,20 @@ public Control(string styleSheet) : base()
             Initialize(style);
         }
 
-        internal void ApplyAttributes(View view, ViewStyle viewStyle)
+        /// Internal used.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ICommand Command
```

Comment:
What is the `Command`?

Reply:
It is a request from Mr.Song. I will remove it from this patch